_data/banking.yml

```
@@ -698,12 +874,16 @@ websites:
       - proprietary
       - hardware
     doc: https://www.hypovereinsbank.de/hvb/services/digitales-banking/psd2/tan-verfahren
+    regions:
+      - de
 
   - name: IberiaBank
     url: https://www.iberiabank.com
     twitter: IBERIABANK
     facebook: IberiaBank
```

```suggestion
  - name: IBERIABANK
    url: https://www.iberiabank.com
    twitter: IBERIABANK
    facebook: IBERIABANK
```
Update Facebook and stylization of name.